src/GeositeFramework/js/Plugin.js

```diff
@@ -57,7 +57,7 @@ require(['use!Geosite',
                                 });
                             }
                             d.resolve();
-                            esriMap.resize(true);
+                            //esriMap.resize(true);
```

Comment:
Sure. I'm adding a `TODO` comment plus a short explanation of why it's commented out & a link to the docs for reference.